staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -264,7 +296,7 @@ func (r *crdHandler) removeDeadStorage() {
 		}
 		if !found {
 			glog.V(4).Infof("Removing dead CRD storage for %v", s.requestScope.Resource)
-			s.storage.DestroyFunc()
+			s.storage.CustomResource.DestroyFunc()
```

Comment:
// destroy only the main storage. Those for the subresources share cacher and etcd clients.